keyboards/cipulot/ec_tkl_x/keyboard.json

```diff
@@ -0,0 +1,714 @@
+{
+    "manufacturer": "Cipulot",
+    "keyboard_name": "EC TKL X",
+    "maintainer": "Cipulot",
+    "bootloader": "stm32-dfu",
+    "build": {
+        "lto": true
+    },
+    "diode_direction": "COL2ROW",
+    "eeprom": {
+        "wear_leveling": {
+            "backing_size": 8192
+        }
+    },
+    "features": {
+        "bootmagic": false,
+        "console": true,
+        "extrakey": true,
+        "mousekey": true,
+        "nkro": true,
+        "rgblight": true
+    },
+    "processor": "STM32F411",
+    "qmk": {
+        "locking": {
+            "enabled": true,
+            "resync": true
+        }
+    },
+    "rgblight": {
+        "animations": {
+            "alternating": true,
+            "breathing": true,
+            "christmas": true,
+            "knight": true,
+            "rainbow_mood": true,
+            "rainbow_swirl": true,
+            "rgb_test": true,
```

Comment:
Are you sure that you want this enabled?

Reply:
If you're referring to the RGB test then yes, I use it for validating at a quick glance that the RGB packages are good and didn't suffer heat issues during manufacturing.
Having the RGB cycle by itself instead of me having to select R, G and B in sequence is faster.